app/uploaders/image_uploader.rb

```diff
@@ -0,0 +1,51 @@
+class ImageUploader < CarrierWave::Uploader::Base
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
```

Comment:
Do you need to set it here? I think it should be handled in https://github.com/IF-115-Ruby/OSBB/blob/development/config/initializers/carrierwave.rb